### ISSUE TITLE
Loading state_dict of dual_schedulers

### ISSUE THREAD

**Issue by juan43ramirez** [cooper-org/cooper#43]
## Bug

The instantiation of the `dual_scheduler` inside  `ConstrainedOptimizer.load_from_state_dict` method has a bug. The variable containing the dual scheduler class is called `dual_scheduler_class` as opposed to `dual_scheduler`.
https://github.com/cooper-org/cooper/blob/f75dff39608f2d2895411c6e40e3675aba03c0f2/cooper/constrained_optimizer.py#L491

## Steps

Replace current line to `dual_scheduler = dual_scheduler_class(dual_optimizer)`

## Expected behavior

This should enable the loading of a `dual_scheduler` from a checkpoint.

**Comment by juan43ramirez** [comment]
Closed by #44
